modules/understanding-upgrade-channels.adoc

```diff
@@ -7,17 +7,156 @@
-4.4 release. This ensures administrators make an explicit decision to upgrade to
-the next minor version of {product-title}. Channels only control updates and
-have no impact on the version of the cluster you install; the
+= {product-title} Upgrade Channels and Releases
+
+In {product-title} 4.1, Red Hat introduced the concept of Upgrade Channels for
+recommending the appropriate release versions for cluster upgrade. Upgrade Channels
+separate upgrade strategies and are also used to control the cadence of upgrades.
+Upgrade Channels are tied to a minor version of {product-title}. For instance,
+{product-title} {product-version} Upgrade Channels will never include an upgrade to a
+4.5 release. This ensures administrators make an explicit decision to upgrade to
+the next minor version of {product-title}. Upgrade Channels only control release
+selection and have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades phased roll out] for more information on the types of updates and upgrade channels.
+{product-title} {product-version} has the following Upgrade Channels to choose from:
+
+.  candidate-{product-version}
+.  fast-{product-version}
+.  stable-{product-version}
+
+= candidate-{product-version}
+
+The candidate-{product-version} channel will contain all z-stream ({product-version}.z)
+releases. If a given release is only available in the candidate-{product-version}
+channel and not available in either the fast-{product-version} or stable-{product-version}
+channels, it is considered a release candidate. Although release candidates contain
+all the features of the product, they are not supported and should only be used to
+test feature acceptance and assist in qualifying the next version of 
+{product-title} in your infrastructure. Additionally, a release candidate can only be
+upgraded to another release candidate version and, if one does not exist, is not
+upgradable.
+
+The candidate-{product-version} channel also allows upgrading from a previous minor version of
+{product-title} to the current release candidate version.
+
++                                                                                    
+[NOTE]                                                                               
+==== 
+Release candidates differ from the nightly builds found on try.openshift.com. Nightly
+builds are available for early access to features but updating to or from nightly
+builds is neither recommended nor supported. Nightly builds are not avilable in
+any Upgrade Channels.
+==== 
+
+= fast-{product-version}
+
+The fast-{product-version} channel is updated with new {product-version} patch
+versions as soon as Red Hat declares the given patch as a general availability
+release. As such, these releases are fully supported, are production quality and have
```

Comment:
nit: do we have a [serial comma][1] docs policy?

[1]: https://en.wikipedia.org/wiki/Serial_comma

Reply:
@kalexand-rh do we have a [serial comma](https://en.wikipedia.org/wiki/Serial_comma) docs policy?

Reply:
No guidance provided so left as is.